templates/20years/index.html

```diff
@@ -0,0 +1,1423 @@
+{% extends "20years/base_20years.html" %}
+
+{% block title %}20 years of Canonical Ubuntu{% endblock %}
+
+{% block meta_description %}
+  We're celebrating 2 decades of innovation and charting a course for our future. And we want you to celebrate with us.
+{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1vlENASSxNJdnK5tRuNYEZGVbh_qrVN0lhon8SOYo6A0/edit
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+
+  <section class="p-strip is-shallow">
+    <div class="row--50-50-on-large">
+      <div class="col">
+        <h1>
+          <span class="u-off-screen">20 years of Ubuntu</span>
```

Comment:
The softest of suggestions, just seems a bit closer to what sighted users would read from this heading. Feel free to ignore if you think it's not really needed
```suggestion
          <span class="u-off-screen">20 years of Ubuntu is just the beginning</span>
```